quic/s2n-quic-core/src/event/s2n_quic.bt

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+ #include "quic/s2n-quic-core/src/event/generated_s2n_quic_bpf_events.h"
+
+BEGIN {
+  printf("s2n-quic");
+}
+
+usdt:./target/debug/s2n-quic-qns:on_connection_started
+{
+    /* arg0 is `id` for connection level events */
+    printf("starting connection %d\n", arg0);
+    @cs++;
+}
+
+usdt:./target/debug/s2n-quic-qns:on_connection_closed
```

Comment:
I guess we expect s2n-quic to both be used as a static dependency and as a shared library. It seems that in the former case, we have to target the individual programs that statically link s2n-quic.